projenrc/next-version.ts

```diff
@@ -0,0 +1,16 @@
+import { schemasChanged } from './update-schema';
+
+/**
+ * If any of the schema files changed, we need to bump the major version
+ */
+async function main() {
+  const latestTag = process.env.LATEST_TAG;
+  if (latestTag && schemasChanged(latestTag)) {
```

Comment:
Why do we can about the existence of `LATEST_TAG` here?

Reply:
Otherwise this script wouldn't work for the very first release (there wouldn't be a "latest" tag)

Reply:
Completely spaced out, sorry 👍